src/main/java/seedu/address/model/person/Person.java

```diff
@@ -26,6 +28,23 @@ public class Person {
     private final Address address;
     private final Set<Tag> tags = new HashSet<>();
 
+    private final List<Transaction> transactions = new ArrayList<>();
+
+    /**
+     * Every field must be present and not null.
+     */
+    public Person(Name name, Company company, Phone phone, Email email, Address address, Set<Tag> tags,
+                  List<Transaction> transactions) {
+        requireAllNonNull(name, phone, email, address, tags, transactions);
```

Comment:
I don't think transactions should be included in the Person constructor. Because the add Person command doesn't have a transaction field.